hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -148,22 +145,17 @@ public OperationServiceImpl(NodeEngineImpl nodeEngine) {
         this.nodeEngine = nodeEngine;
         this.node = nodeEngine.getNode();
         this.thisAddress = node.getThisAddress();
-        this.logger = node.getLogger(OperationService.class);
+        this.logger = node.loggingService.getLogger(OperationService.class);
```

Comment:
`Node#getLogger(java.lang.Class)` will do the same thing, forward the method call to `LoggingServiceImpl#getLogger(java.lang.Class)`. Is this change deliberate then? Should other calls to `Node#getLogger(java.lang.Class)` in this class be changed too?